openwisp_radius/tasks.py

```diff
@@ -39,3 +51,41 @@ def delete_unverified_users(older_than_days=1, exclude_methods=''):
 @shared_task
 def convert_called_station_id(unique_id=None):
     management.call_command('convert_called_station_id', unique_id=unique_id)
+
+
+@shared_task
+def send_login_email(accounting_data):
+    User = get_user_model()
+    Organization = swapper.load_model('openwisp_users', 'Organization')
+    username = accounting_data.get('username', None)
+    org_uuid = accounting_data.get('organization')
+    try:
+        user = User.objects.get(username=username)
+        organization = Organization.objects.get(id=org_uuid)
+        if organization.is_member(user):
+            from sesame.utils import get_query_string
+
+            org_radius_settings = organization.radius_settings
+            login_url = org_radius_settings.login_url
+            if login_url:
```

Comment:
let's do:

```python
if not login_url:
    return
```

To avoid unnecessary indentation which makes this harder to read.